ruby/vampires.rb

```diff
@@ -0,0 +1,139 @@
+#Initialize required variables
+full_name = ""
+age = 0
+birth_year = 0
+correct_age = false
+allergy = ""
+serve_garlic = false
+health_insurance = false
+num_employees = 0
+
+#Compares age to birth year to see if age is accurate
+def compare_age(age, birth_year)
+  if (2017 - age) == birth_year
+    return true
+  else
+    return false
+  end
+end
+
+#Takes in all variable to determine likelihood of being a vampire and prints result
+def determine_vampire(name, correct_age, serve_garlic, health_insurance, employee_num, allergy)
```

Comment:
Great work encapsulating with methods.  I think that you could break this down a little more.  We want out methods to use separation of concerns.  This means that each method should do only one thing essentially.

How about a method that returns the scale value and a method the prints to result for a refactor point.